README.md

```diff
@@ -60,38 +70,37 @@ echidna_convert: failed!💥
     mint(57896044618658097711785492504343953926634992332820282019728792003956564819968)
 ```
 
+Echidna supports testing complex contracts with two features. First, it [allows initializing a fuzzing campaign with arbitrary transactions using Etheno and Truffle](https://github.com/crytic/echidna/wiki/Deployment-of-a-contract-using-Truffle,-Ganache-and-Etheno-to-test-with-Echidna). Second, it can test more than one contract at the same time, calling any public or external function of any tested contract. Use `multi-abi: true` in your configuration file to allow Echidna to generate transactions to call more than one contract at the same time.
```

Comment:
again let's be more clear here. we don't test multiple contracts. we only test a single contract but we can call any contract we have an abi for.